bincrafters_conventions/actions/update_other_travis_to_ci_dir_name.py

```diff
@@ -1,4 +1,4 @@
-import shutil
+# import shutil
```

Comment:
please, don't commit commented code, we can reflog if necessary.

Reply:
Removed